### DISCUSSION TITLE
委托来了

### DISCUSSION THREAD

**Discussion by DJLFi** [share121/inter-knot#2686] [其他]
怎么才能快速升级绳网啊？
![image_download_1723798247455](https://github.com/user-attachments/assets/9edd2926-e285-4b88-bb92-e78c7874de22)

**Comment by CloudSwordSage** [comment]
建公开仓库，fork来的仓库也算<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
